apache/zeppelin
# [ZEPPELIN-1056] Embed webfont when building

### What is this PR for?
This is some changes to include the previously webfonts when building Zeppelin.
If there is too much problem building (In the case of China), we could eventually make the fonts part of the source code


### What type of PR is it?
Improvement

### Todos
* [x] - Add the fonts licenses (Apache 2.0 + SIL Open Font License v1.10)

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1056

### How should this be tested?
Deactivate the fonts on your computer, build and launch Zeppelin.
The Zeppelin logo should have the Patua One font

### Questions:
* Does the licenses files need update? Yes
* Is there breaking changes for older versions? No
* Does this needs documentation? No
